### PR TITLE
Author id for posts & test suite update

### PR DESCRIPTION
**Pull request by GilgusMaximus**
1) The test suit now contains a test for an upcoming video. The video will be premiering in 2024. So that should keep the test running for some time.

If it fails at some point, just update the video to a new one.

2) I added the authorid to all community posts and to the video and playlist elements, as they were missing and one could not open channel links to youtube then.

All tested, except the playlist author id as I could not find one channel which has a playlist posted recently. To be tested, otherwise I would keep it in the wild until someone stumbles upon one and it fails (or it might work even, as I inferred that it would work like the other elements).

### CODE REVIEW

File: test/channelVideos.test.js [hunk 1]
```diff
@@ -45,4 +45,11 @@ describe('Getting channel videos', () => {
       expect(data.items.length).toBe(0)
     })
   })
+
+  test('Upcoming video', () => {
+    const parameters = { channelId: 'UCUKPG1-r4WFyxpyhIuCs86Q', channelIdType: 1 }
```
Comment (PikachuEXE) [inline test/channelVideos.test.js:50]:
```suggestion
    // https://www.youtube.com/channel/UCUKPG1-r4WFyxpyhIuCs86Q
    // This channel has a video premiering in 2024/3/31
    const parameters = { channelId: 'UCUKPG1-r4WFyxpyhIuCs86Q', channelIdType: 1 }
```

**Review by PikachuEXE — CHANGES_REQUESTED** [review]

**Comment by GilgusMaximus** [comment]
Good point to add the comment

**Review by PikachuEXE — DISMISSED** [review]

**Review by ChunkyProgrammer — DISMISSED** [review]

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@absidue is it possible for u to review this. Im afk for a few days

File: package-lock.json [hunk 2]
```diff
@@ -1,6074 +1,113 @@
 {
   "name": "yt-channel-info",
   "version": "3.0.4",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```
Comment (absidue) [inline package-lock.json:4]: Is this version downgrade intentional?

Comment (PikachuEXE) [reply package-lock.json:4]: Oops I missed that
Might be easier to just use yarn on all FT repos...

Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply package-lock.json:4]: @GilgusMaximus

Comment (ChunkyProgrammer) [reply package-lock.json]: I removed the packagelock change

**Review by PikachuEXE — APPROVED** [review]

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — APPROVED** [review]

**Review by absidue — APPROVED** [review]

**Pull request merged**